dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -4193,11 +4190,11 @@ private RegressionTestScenario(string testPagename, bool isLeftnavTest, bool isU
         public bool IsUsingRS4Style { get; private set; }
         public static List<RegressionTestScenario> BuildLeftNavRegressionTestScenarios()
         {
-            return BuildTestScenarios(RegressionTestType.LeftNav | RegressionTestType.LeftNavRS4);
+            return BuildTestScenarios(RegressionTestType.LeftNav);
         }
         public static List<RegressionTestScenario> BuildAllRegressionTestScenarios()
         {
-            return BuildTestScenarios(RegressionTestType.LeftNav | RegressionTestType.LeftNavRS4 | RegressionTestType.TopNav);
+            return BuildTestScenarios(RegressionTestType.LeftNav | RegressionTestType.TopNav);
```

Comment:
Does something need to be tested specific for RS4 that is not needed anymore ?

Reply:
There were separate test pages that were applying older OS control templates to navigation view. However, that was stale because it is not reflective of what is going on anymore (as we removed os specific themeresource files for the most part). I removed the parts of the downlevel tests that were specifically testing a part of the older control template, however I kept the downlevel tests that were actually testing behavior that was not dependent directly on the older template version.